src/foam/u2/view/EnumLegendView.js

```diff
@@ -50,7 +50,7 @@ foam.CLASS({
           statusEnum => {
             return this.E().start().addClass(this.myClass('eachValue'))
               .start().add(
-                foam.u2.view.ReadOnlyEnumView.create({
+                foam.u2.view.EnumBadgeView.create({
```

Comment:
Doesn't EnumBadgeView just become the new ReadOnlyEnumView? Couldn't it just take its place?

Reply:
was unsure of what context this view was used in so I left it that way. Replaced now